rollup/client.go

```diff
@@ -42,6 +42,10 @@ type SyncStatus struct {
 	CurrentTransactionIndex      uint64 `json:"currentTransactionIndex"`
 }
 
+type L1GasPrice struct {
+	GasPrice hexutil.Big `json:"gasPrice"`
```

Comment:
I don't think `hexutil.Big` will deserialize the value correctly as it is returned in decimal as a string in units of wei.
See:
https://github.com/ethereum-optimism/go-ethereum/blob/b54a387538c9272274ad38ae96c3951a2f652571/common/hexutil/json.go#L160
https://github.com/ethereum-optimism/data-transport-layer/commit/a01d4c1c74804fd4f53ff87107e164b83a7386eb#diff-70ed2f7cc4c97ed88c9a89fbedc896eb1abefe87887ee26731f393abaa20cfdaR186

One solution would be to turn it into  `uint64` with `string` in the struct tag and you could make a getter `L1GasPrice.Big` or something like that
```golang
type L1GasPrice struct {
   GasPrice uint64 `json:"gasPrice,string"
}
```

Another solution would be making the HTTP endpoint return a hex string, but that makes it less intuitive for humans